pkg/mac/build-functions.sh

```
@@ -457,6 +457,68 @@ _prune_dangling_symlinks() {
+    fi
+
+    local found="" f deps
+    while IFS= read -r f; do
+        # otool prints the filename header then one line per dependency; drop
+        # the header, take the install-name column, and keep only build-host
+        # paths. grep exits non-zero (no match) for a clean binary, which is
+        # the common case, so fall through to the next file.
+        deps=$(otool -L "${f}" 2>/dev/null | tail -n +2 | awk '{print $1}' | \
+            grep -E "^(${PREFIXES})") || continue
+        echo "ERROR: ${f} links against build-host libraries:" >&2
+        echo "${deps}" | sed 's/^/    /' >&2
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)